test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,20 @@
+package codeu.chat.util
+
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public final class TokenizerTest{
+  public void testWithQuotes() throws IOException{
+    final Tokenizer tokenizer = new Tokenizer("\"hello world\"");
+    assertEquals(tokenizer.next(), "hello world");
+    assertEquals(tokenizer.next(), null);
+  }
+  
+  public void testWithNoQuotes throws IOException{
```

Comment:
This function definition is missing parenthesis, which is keeping your branch from compiling